crates/common/src/forking.rs

```diff
@@ -194,7 +194,7 @@ pub async fn fork_and_prepare(
         .modify_cfg_chained(|c| {
             c.chain_id = chain_id;
             c.spec = spec_id;
-            c.disable_nonce_check = quick; // Disable nonce check in quick mode
+            c.disable_nonce_check = true; // Disable nonce check in quick mode
```

Comment:
I think we should keep the nonce check? 

The nonce issue is rooted in those ROC servers, not in EDB, from my point of view.

Alternatively, if the nonce issue is indeed annoying, we can add a new cli arg to relax checks?

Reply:
Ops, that was a mistake